src/rdiff-backup-delete

```diff
@@ -25,13 +25,17 @@
 #
 
 import os
+import re
 import sys
 
+from distutils import spawn
```

Comment:
See https://github.com/rdiff-backup/rdiff-backup/blob/master/docs/CODING.md#import

Reply:
Be more specific here. I have no clue what is the problem. `spawn` is a module.

Reply:
Ups. Sorry, my bad. You're right.